### PR TITLE
Error message and minor fixes, issue #447

### PR DESCRIPTION
**Pull request by namsaraeva**
Potential (and temporary) fix for the issue #447. 

As discussed with @Zethson, we should error out the case when multiple reference values are provided, but no additional information, e.g. sex/age/ethnicity. 

I added some code to check if additional columns like sex/age/ethnicity were passed as arguments (at least one is needed to make the value True). Next, if this value is False (no additional column passed) and more than 1 reference value was detected, ValueError is thrown with the message to add additional information. 

Not sure how to make it more specific and flexible, since it is theoretically possible that dataframes other than laposata will be passed onto this method and columns other than sex/age/ethnicity are possible as well.

I also did not really catch what is the difference between our laposata dataframe and the one @xinyuejohn mentioned recently (https://accessmedicine.mhmedical.com/content.aspx?bookid=1069&sectionid=60775149)? Visually they are the same to me, however we have a Sex column.  

Generally, I think we should somehow re-design this method and make it more flexible for other covariates... but that's a big goal. 

Eager to discuss other possible solutions with @Zethson and @xinyuejohn

### CODE REVIEW

File: ehrapy/preprocessing/_quality_control.py [hunk 1]
```diff
@@ -263,13 +263,13 @@ def qc_lab_measurements(
         adata: Annotated data matrix.
         reference_table: A custom DataFrame with reference values. Defaults to the laposata table if not specified.
         measurements: A list of measurements to check.
-        unit: The unit of the measurements. Defaults to SI .
+        unit: The unit of the measurements. Defaults to 'traditional'.
```
Comment (Zethson) [inline ehrapy/preprocessing/_quality_control.py:266]: Hmm, should this be the default? Is it really more common?

Comment (Zethson) [reply ehrapy/preprocessing/_quality_control.py:266]: And is it true? :joy:

**Review by Zethson — CHANGES_REQUESTED** [review]
@namsaraeva could you add a small test for this case, please?

File: ehrapy/preprocessing/_quality_control.py (hunk 1 above)
Comment (namsaraeva) [reply ehrapy/preprocessing/_quality_control.py:266]: I changed it to `traditional` due to this line: 

`reference_column = "SI Reference Interval" if unit == "SI" else "Traditional Reference Interval"`

But I don't really know what is real default

**Pull request merged**